src/components/ContactsList/index.js

```diff
@@ -24,6 +24,29 @@ export const sortGivenNameFirst = (contact, comparedContact) => {
   return nameA.localeCompare(nameB);
 };
 
+const cozyHTTP = {
+  getContacts: async () => {
+    const contactsRaw = await cozy.client
+      .fetchJSON("GET", "/data/io.cozy.contacts/_all_docs?include_docs=true")
+      .then(data => data.rows)
+      .catch(error => {
+        this.setState(state => ({ ...state, contacts: [], error }));
+      });
+    const contacts = (contactsRaw || [])
+      .map(row => ({
+        ...row.doc
+      }))
+      .filter(contact => contact.email);
+    return contacts;
+  },
+
+  deleteContact: contact =>
+    cozy.client.fetchJSON(
+      "DELETE",
+      `/data/io.cozy.contacts/${contact._id}?rev=${contact._rev}`
+    )
+};
```

Comment:
A small extract/abstract of the API couch that communicate with cozy-stack.

For now, it exposes two functions:

- getContacts: fetch all `io.cozy.contacts`
- deleteContact: delete the given `io.cozy.contacts`